.github/workflows/main.yml

```diff
@@ -55,7 +56,7 @@ jobs:
       - run: |
           v='${{ matrix.config.crystal }}'
           crystal --version | grep -E "${v}\\b"
-        if: ${{ contains(matrix.config.crystal, '.') }}
+        if: ${{ contains(matrix.config.crystal, '.') && !contains(matrix.config.crystal, 'branch:') }}
```

Comment:
Changes should be exactly duplicated to `.github/workflows/release.yml`. Although it is expected to fail at first, of course.